File: velolabs_repos/lattis_android/app/src/main/java/com/lattis/ellipse/data/network/model/response/history/RideHistoryResponse.java
package com.lattis.ellipse.data.network.model.response.history;

import com.google.gson.annotations.SerializedName;
import com.lattis.ellipse.data.network.model.response.AbstractApiResponse;

import java.util.List;

/**
 * Created by ssd3 on 8/16/17.
 */

public class RideHistoryResponse extends AbstractApiResponse {

    @SerializedName("payload")
    List<RideHistoryDataResponse> rideHistoryPayloadResponses;

    public List<RideHistoryDataResponse> getRideHistoryDataResponse() {
        return rideHistoryPayloadResponses;
    }
}

File: velolabs_repos/lattis_android_v2/data/src/main/java/com/lattis/data/entity/response/history/RideHistoryResponse.kt
package com.lattis.data.entity.response.history

import com.google.gson.annotations.SerializedName
import com.lattis.data.entity.response.base.AbstractApiResponse

class RideHistoryResponse : AbstractApiResponse() {
    @SerializedName("payload")
    var rideHistoryDataResponse: List<RideHistoryDataResponse>? = null
}